src/vcpkg/commands.remove.cpp

```diff
@@ -99,66 +99,49 @@ namespace vcpkg::Remove
             spgh.state = InstallState::NOT_INSTALLED;
             write_update(fs, installed, spgh);
 
-            status_db->insert(std::make_unique<StatusParagraph>(std::move(spgh)));
+            status_db.insert(std::make_unique<StatusParagraph>(std::move(spgh)));
         }
     }
 
-    static void print_plan(const std::map<RemovePlanType, std::vector<const RemovePlanAction*>>& group_by_plan_type)
+    constexpr struct AddressOf
     {
-        static constexpr std::array<RemovePlanType, 2> ORDER = {RemovePlanType::NOT_INSTALLED, RemovePlanType::REMOVE};
-
-        for (const RemovePlanType plan_type : ORDER)
+        template<class T>
+        const T* operator()(const T& t) const
         {
-            const auto it = group_by_plan_type.find(plan_type);
-            if (it == group_by_plan_type.cend())
-            {
-                continue;
-            }
+            return &t;
+        }
```

Comment:
I don't believe this overload does anything. (T can be deduced to be const something below)